kernel/build/config-amd64

```diff
@@ -5421,11 +5427,11 @@ CONFIG_CRYPTO_CRC32=y
 # Compression
 #
 CONFIG_CRYPTO_DEFLATE=y
-# CONFIG_CRYPTO_LZO is not set
+CONFIG_CRYPTO_LZO=y
```

Comment:
i guess these cannot be built as modules?

Reply:
I chose to make it a builtin, as it is also a builtin in the arm64 kernel.

Reply:
Also I wasn't sure if I need to put them into its own dedicated package or if it's fine to ship it with the btrfs one.

Reply:
One issue comes up if build `CONFIG_LZO_DECOMPRESS=m` on amd64:

This option is already a builtin on arm64, therefore the list of modules to package differs on arm64 vs amd64.

* option 1: make it a builtin on both platforms
* option 2: is there a way to have platform specific module lists?

Reply:
the diff is probably an oversight, we can make it similar, if it's a dependency for btrfs and can be built as a module, let's do that

Reply:
something is using LZO in the arm-kernel... I don't have proper tooling to investigate, but if I apply the values from amd64 to arm64 and run `make oldconfig`, they get reenabled to `=y`.

Reply:
applied.